sdk/remoterendering/azure-mixedreality-remoterendering/azure/mixedreality/remoterendering/_remote_rendering_client.py

```diff
@@ -78,20 +78,25 @@ def __init__(self, endpoint, account_id, account_domain, credential, **kwargs):
         if not credential:
             raise ValueError("credential cannot be None")
 
+        self.polling_interval = kwargs.pop("polling_interval", 5)
+        endpoint_url = kwargs.pop('authentication_endpoint_url', construct_endpoint_url(account_domain))
+
+        def build_credentials(input_credentials):
+            # type: (Any) -> Any
+            return get_mixedreality_credential(
+                account_id=account_id,
+                account_domain=account_domain,
+                credential=input_credentials,
+                endpoint_url=endpoint_url)
+
         if isinstance(credential, AccessToken):
-            cred = StaticAccessTokenCredential(credential)  # type: TokenCredential
+            pipeline_credential = build_credentials(
+                StaticAccessTokenCredential(credential))  # type: TokenCredential
```

Comment:
I found

```py
def get_mixedreality_credential(account_id, account_domain, endpoint_url, credential, **kwargs):
    # type: (str, str, str, TokenCredential, Any) -> TokenCredential
    if isinstance(credential, StaticAccessTokenCredential):
        return credential

    return MixedRealityTokenCredential(
        account_id=account_id,
        account_domain=account_domain,
        endpoint_url=endpoint_url,
        credential=credential,
        **kwargs)
```

So here it is actually

cred = StaticAccessTokenCredential(credential)  # type: TokenCredential?

Reply:
Because I did not know about type hinting (I'm not using python very often, forgive me).

Using the method helped the pylint and mypy to get that cred was type Any. Of course this is a bit overkill just for that, so I changed it back.